memcache/memcache.go

```diff
@@ -90,14 +105,34 @@ const (
 
 // protoToItem converts a protocol buffer item to a Go struct.
 func protoToItem(p *pb.MemcacheGetResponse_Item) *Item {
+	if p.IsDeleteLocked != nil && *p.IsDeleteLocked {
+		// "delete lock" for a duration is not a feature available in the Go lib.
+		// Such items may exist in memcache though, e.g. created by the Java lib.
+		// In this case, nil is more appropriate than an item with empty value.
+		// For a single Get, nil will translate to ErrCacheMiss.
+		return nil
+	}
 	return &Item{
 		Key:   string(p.Key),
 		Value: p.Value,
 		Flags: p.GetFlags(),
 		casID: p.GetCasId(),
+		Timestamps: ItemTimestamps{
+			Expiration: timeSecToTime(p.Timestamps.GetExpirationTimeSec()),
```

Comment:
`p.GetTimestamps()`  instead of `p.Timestamps` to keep it consistent.

Reply:
Ack, thanks.